### PR TITLE
added new langauges

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
Not ready for review. I still need to confirm language codes with Reeza and also add po files.

**Comment by g1039** [comment]
@miltontony @saeedmb8 Ready for review

File: babycenter/settings/base.py [hunk 1]
```diff
@@ -185,8 +187,175 @@
...
+        'bidi': False,
+        'code': 'ku',
+        'name': 'Kurdish',
+        'name_local': 'Kurdî'
+    },
+    'arc': {
+        'bidi': False,
+        'code': 'arc',
+        'name': 'Aramaic',
+        'name_local': 'ܐܪܡܝܐ‎'
```
Comment (miltontony) [inline babycenter/settings/base.py:345]: This doesn't look right

**Comment by smarzban** [comment]
👍

**Pull request merged**